README.md

```diff
@@ -42,8 +42,8 @@ All Saleor services started from a single repository
 
 
 ## How to update the subprojects into newest versions?
-By default, each of the subprojects is bound to particular commit.
-In order to update all of them to their newest master versions, run `git submodule update --remote`
+By default, this repository contains newest current (stable) versions. When new release appear, pull new version of this repository.
```

Comment:
It's a bit sketchy, due to some reasons this doesn't contain stable versions for now (dev dockerfiles added after last stable for dashboard and storefront). I've created a reminder issue in this repo to update subprojects when we'll release newest stable. Not sure if we should merge this part for now.

Reply:
@maarcingebala when do we plan to announce new version?

Reply:
Alternative solution would be creating patches for a last stable version and just bump minor version number.

Reply:
We don't know the release date yet.